tomopy/io/exchange.py

```diff
@@ -83,7 +84,25 @@
            'read_petraIII_p05',
            'read_sls_tomcat']
 
-
+def deprecated(func):
```

Comment:
I think this function should be placed somewhere inside util folder, since it's useful to any deprecated function in the future. But not sure where we should put it? ```util/misc.py```?